### PR TITLE
Document shell completion

### PR DESCRIPTION
**Pull request by catskul**
Add in a stanza to the Usage section about tab completion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -517,3 +518,16 @@ Commands:
   worklog list:        Prints the worklog data for given issue
 
 ```
+
+#### Setting up TAB completion
```
Comment (coryb) [inline README.md]: This wont match your #tab-completion anchor.  GH Markdown is not that clever.  So you can change the TOC to `#setting-up-tab-completion` or change the topic to `#### TAB completion`.

Comment (catskul) [reply README.md]: Good catch. Fixed!

**Comment by coryb** [comment]
Thanks for the help!

**Comment by catskul** [comment]
No problem. So glad to have a usable tool that keeps me away from the browser. Thanks for making this tool!

**Pull request merged**